data/tlg0062/tlg054/tlg0062.tlg054.perseus-eng3.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">The Dipsads</title>
+<title xml:lang="grc">Περὶ τῶν διψάδων</title>
+<author>Lucian of Samosata</author>
+<editor role="translator">Kenneth Kilburn</editor>
+<sponsor>Perseus Project, Tufts University</sponsor>
+<principal>Gregory Crane</principal>
+<respStmt>
+<resp>Prepared under the supervision of</resp>
+<name>Gregory Crane</name>
+<name>Lisa Cerrato</name>
+
+</respStmt>
+<funder n="org:NEH">The National Endowment for the Humanities</funder>
+</titleStmt>
+<extent/>
+<publicationStmt>
+<publisher>Trustees of Tufts University</publisher>
+<pubPlace>Medford, MA</pubPlace>
+<authority>Perseus Project</authority>
+    <date type="release" when="2023-06-23">Perseus 5.0</date>
+    <idno type="filename">tlg0062.tlg053.perseus-grc3.xml</idno>
+    <availability>
+        <licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available under a Creative Commons Attribution-ShareAlike 4.0 International License</licence>
+    </availability>
+</publicationStmt>
+
+    <sourceDesc>
+        <listBibl>
+            <biblStruct>
+                <monogr>
+                    <author>Lucian of Samosata</author>
+                    <title xml:lang="eng">Lucian with an English Translation</title>
+                    <editor role="translator">Kenneth Kilburn</editor>
+                    <imprint>
+                        <pubPlace>London</pubPlace>
+                        <publisher>William Heinemann Ltd.</publisher>
+                        <pubPlace>Cambridge, MA</pubPlace>
+                        <publisher>Harvard University Press</publisher>
+                        <date>1959</date>
+                    </imprint>
+                    <biblScope unit="volume">6</biblScope>
+                </monogr>
+<ref target="https://archive.org/details/luciankilb06luciuoft/page/77/mode/2up">Internet Archive</ref>
```

Comment:
I'd in general just stick with one URL (typically the internet archive) but you certainly don't need to remove it.

Reply:
Who knows how long the Internet Archive will be there? Not as long as these files.

Reply:
We tend to use the Internet Archive when we can simply because it means users outside the United States have better access. The HathiTrust restricts user access from outside the United States much more aggressively even for items in the public domain.